code/scripting/api/objs/weaponclass.cpp

```diff
@@ -712,6 +712,36 @@ ADE_VIRTVAR(BurstDelay, l_Weaponclass, "number", "The time in seconds between sh
 	return ade_set_args(L, "f", Weapon_info[idx].burst_delay);
 }
 
+ADE_VIRTVAR(FieldOfFire, l_Weaponclass, "number", "The angular spread for shots of this weapon.", "number", "Fof in degrees, or 0 if handle is invalid")
```

Comment:
Would be good to specify that these two virtvars are in radians

Reply:
They do both specify their units are degrees, if that's what you mean.

Reply:
*surprised pikachu face*

You're right, they do.  Not only that, the angle is treated as degrees in the call to `vm_vec_random_cone`.  Ok then.

Although this does bring to light an unrelated bug.  I'll submit a PR momentarily.